pyhik/hikvision.py

```diff
@@ -430,6 +432,182 @@ def get_device_info(self):
             _LOGGING.error('There was a problem: %s', err)
             return None
 
+    """
+    This function is added by Ayush Pratap Singh (ayushs56@gmail.com)
```

Comment:
Attribution comments aren't needed.  You'll be the owner on the merged PR.

Reply:
Removed attribution comment